x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/rules_table/bulk_actions.tsx

```diff
@@ -53,9 +58,23 @@ export const BulkActions: React.FC<BulkActionsProps> = React.memo(
             </EuiButtonEmpty>
           </EuiFlexItem>
         ) : null}
+        {showRetryFailedRulesButton ? (
```

Comment:
NIT: Sometimes it's more convenient to use the `&&` short-circuit evaluation instead of ternaries with `null` "else cases" at the end.
```ts
  {showComponent && (
    <Component />)
  }
```